src/assets/scripts/client/aircraft/AircraftCollection.js

```diff
@@ -0,0 +1,114 @@
+import _find from 'lodash/find';
+import _isArray from 'lodash/isArray';
+import _isEmpty from 'lodash/isEmpty';
+import _map from 'lodash/map';
+import BaseCollection from '../base/BaseCollection';
+import AircraftTypeDefinitionModel from './AircraftTypeDefinitionModel';
+import { airlineNameAndFleetHelper } from '../airline/airlineHelpers';
+
+/**
+ * Collection of `AircraftInstanceModel` objects
+ *
+ * Responsible for creating new `AircraftInstanceModel` objects when a spawnInterval
+ * fires its `createAircraftWithSpawnPatternModel` callback.
+ *
+ * This collection also keeps a list of `AircraftTypeDefinitionModel` objects, which define each
+ * aircraft type.
+ *
+ * @class AircraftCollection
+ * @extends BaseCollection
+ */
+/* istanbul ignore next */
+export default class AircraftCollection extends BaseCollection {
```

Comment:
This is really more an `AircraftTypeCollection`. we aren't dealing with any actual `AircraftInstanceModels` here.